src/envoy/auth/controller.cc

```diff
@@ -0,0 +1,175 @@
+/* Copyright 2017 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "controller.h"
```

Comment:
src/envoy/auth/controller.h

Reply:
For the header files in the same folder,  it is better to just use file name, not with path.  It is easier to move if desired.

Reply:
No, that is not allowed in C++ style guide.